root/static/scripts/common/components/DescriptiveLink.js

```diff
@@ -24,7 +25,7 @@ const DescriptiveLink = ({entity, content}) => {
     return l('{entity} by {artist}', {
       __react: 'frag',
       entity: link,
-      artist: <ArtistCreditLink artistCredit={entity.artistCredit} key={1} />,
+      artist: <ArtistCreditLink artistCredit={ko.unwrap(entity.artistCredit)} key={1} />,
```

Comment:
Some places this is an observable (releases/tracks in the release editor), some places it's not (recordings, etc.). So we need to pass it to ko.unwrap just it case. We do this for a few other variables below.